trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py

```diff
@@ -53,11 +53,9 @@ def compressibility_drag_wing(state,settings,geometry):
     
     wing = geometry
```

Comment:
Is the geometry that is passed a wing? Or is it the full vehicle?

Is this run for each wing?

Reply:
Looks like it's run for each wing. See in [Fidelity_Zero](https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Analyses/Aerodynamics/Fidelity_Zero.py#L107).